doc/en/conf.py

```diff
@@ -247,7 +247,7 @@
 html_domain_indices = True
 
 # If false, no index is generated.
-html_use_index = True
+html_use_index = False
```

Comment:
Do we really want to disable it entirely? It seems useful to keep even if we don't link to it from the sidebar, no?

Reply:
When would one want to use such an index? IMHO there are better options for all usecases:

- If you know the term you are looking for, it's simpler to type it into the search field.
- If you don't know the term and rather want to see what is there, topic-oriented pages like the references and how-tos are a better place to browse compared to a long alphabetical list.

Therefore I suggest to remove it. But if you really want to keep it, we can add that page to the "full table of contents".